docs/_static/styles/custom.css

```diff
@@ -0,0 +1,155 @@
+/*
+  Furo CSS variables
+  https://github.com/pradyunsg/furo/blob/main/src/furo/assets/styles/variables/_index.scss
+  https://github.com/pradyunsg/furo/blob/main/src/furo/theme/partials/_head_css_variables.html
+*/
+:root {
+  /* adjust font-stack by adding "SF Pro Display" and "SF Mono" */
+  --font-stack: -apple-system, BlinkMacSystemFont, SF Pro Display, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
+  --font-stack--monospace: SFMono-Regular, SF Mono, Menlo, Consolas, Monaco, Liberation Mono, Lucida Console, monospace;
+  /* Streamlink colors */
+  --color-brand-primary: rgb(18, 22, 87);   /* dark icon color (shadow icon color: rgb(13, 16, 65)) */
+  --color-brand-content: rgb(0, 115, 189);  /* bright icon color */
+  /* misc */
+  --sidebar-item-spacing-vertical: .4rem;
+  --admonition-font-size: var(--font-size--small);
+  --admonition-title-font-size: var(--font-size--normal);
+}
+
+/*
+  Generic style overrides
+*/
+
+html {
+  /* unset @media (min-width: $full-width + $sidebar-width) query which sets font-size to 110% */
+  font-size: 100% !important;
```

Comment:
Suggestion: don't override this, since that's a pattern followed by many other themes -- it makes the reading experience on larger format displays and ultra-wide monitors better.

Reply:
See https://github.com/squidfunk/mkdocs-material/blob/ef5f2930003d1db53e5f7131ddcfff4046663a23/src/assets/stylesheets/main/layout/_base.scss#L37, for another example (although it's burried in a bunch of hacks).

Reply:
@bastimeyer Do you want to update this?

Reply:
No, because that's an opinionated design choice which I don't share. People using bigger and/or wider screen resolutions can choose to increase their desktop's DPI scaling or font scaling if reading text in the browser's default text size is a problem for them, which they probably already have done, making this media query highly questionable, especially for those others who dislike large texts. Apart from that, all text sizes defined on the default scaling are carefully chosen to match whole pixel values, and setting the base scaling to 110% breaks this logic.

Regarding the overall state of the PR, which you're probably not aware of, there are still some unresolved issues with the theme left, like for example scrollbar issues with both sidebars, which I don't want to fix in a custom workaround here and have therefore opened issue on its issue tracker a few days ago. Once this has all been fixed on the theme side, I will update the PR branch and check the remaining issues like the message box colors and the CLI parameter layout.

Reply:
@bastimeyer I'm aware of some of the outstanding issues per your `to-do` section but I wasn't aware you had opened PRs in the other issue trackers. Thanks for the update.